windows/src/desktop/setup/Keyman.Setup.System.InstallInfo.pas

```diff
@@ -625,4 +626,26 @@ procedure TInstallInfoFileLocation.UpgradeToLocalPath(const ARootPath: string);
   FLocationType := iilLocal;
 end;
 
+
+
+// Pulled from IdGlobalProtocols so we don't have to import it
+// OS-independant version
```

Comment:
```suggestion
// OS-independent version
```

Reply:
It's from the Indy code! I haven't fixed anything except stripping out the huge spaghetti of cross-platform support.